dapps/marketplace/src/hoc/withGrowthRewards.js

```diff
@@ -4,15 +4,16 @@ export default function withGrowthRewards(WrappedComponent) {
   const WithGrowthRewards = props => {
     const ognListingRewards = {}
 
-    if (props.growthCampaigns && props.tokenDecimals) {
+    const tokenDecimals = props.tokenDecimals || 18
+    if (props.growthCampaigns && tokenDecimals) {
```

Comment:
I think tokenDecimals will always have a value now so can probably just make this `if (props.growthCampaigns)`

Reply:
dooh 🤦‍♂ thanks